src/resources/postcss/help-hub-iframe.pcss

```diff
@@ -228,3 +191,13 @@ body[data-opted-in="0"] .docsbot-widget-background {
 		margin-top: 0 !important;
 	}
 }
+
+/* Force Help Scout Beacon to fill the viewport responsively */
+.BeaconContainer {
+    width: 90% !important;
```

Comment:
Use tabs instead of spaces.

I see that this file is full of `!important` tags, so maybe it's unavoidable, but if at all possible I would love to see a more specific selector. Once you start using `!important`, it's hard to get away from it.